src/workerd/api/worker-rpc.c++

```diff
@@ -95,14 +95,11 @@ private:
 
 jsg::Promise<jsg::Value> WorkerRpc::sendWorkerRpc(
     jsg::Lock& js,
+    rpc::JsRpcTarget::Client client,
     kj::StringPtr name,
     const v8::FunctionCallbackInfo<v8::Value>& args) {
-
   auto& ioContext = IoContext::current();
```

Comment:
nit: Should we just pass the `IoContext&` in via method param since we already called this in the caller in `http.c++`?

Reply:
We could, but `IoContext::current()` is not expensive to call, so I didn't think it was worth the API change. It's sort of a coincidence that the call site happens to have its own copy of this reference.